### PR TITLE
Fix/rare segfaults align wave simd

### PR DESCRIPTION
**Pull request by rrahn**
Hopefully,
* fixes issue with ppc64: see #2245
* fixes some rare segfaults.

### CODE REVIEW

File: tests/align_parallel/test_align_wavefront_alignment_scheduler.h [hunk 1]
```diff
@@ -200,7 +200,6 @@ SEQAN_DEFINE_TEST(test_align_parallel_wavefront_alignment_scheduler_async_with_e
         test_align_parallel::RaiiEvent event;
         TInnerTask task = [&]()
         {
-            if (calledIds[id] > 0)
```
Comment (h-2) [inline tests/align_parallel/test_align_wavefront_alignment_scheduler.h]: is this change intended?

Comment (rrahn) [reply tests/align_parallel/test_align_wavefront_alignment_scheduler.h]: yes, because it solves the race condition seen in the PPC test.

**Comment by h-2** [comment]
how is this related to #2247?

**Comment by rrahn** [comment]
???
It is related to the failed PPC test, so it is linked to #2245.

**Comment by h-2** [comment]
This PR it contains #2247. Is that related to this PR or just chance?

**Comment by rrahn** [comment]
ups - probably committed from the wrong local branch.
Then let's close #2247 and use this instead.
